caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -391,9 +391,54 @@
                         {
                           window.location.hash = $(this).find("a").first().attr("href");
                         });
+
+      this._initBackButtonHandling();
+
     };
 
     /**
+     * Ensure back button navigates through tabs user (or app) has clicked through.
```

Comment:
For consistency, please use double quotes for strings.  The single quotes can be used in the DOM output.  Also, use the !== always unless there is a good reason not to.